packages/editor-ui/src/stores/ui.store.ts

```diff
@@ -628,6 +630,40 @@ export const useUIStore = defineStore(STORES.UI, () => {
 		lastCancelledConnectionPosition.value = undefined;
 	}
 
+	const isInstanceOwnerInCloud = () => {
+		const deploymentType = settingsStore.deploymentType;
+		return hasPermission(['instanceOwner']) && deploymentType === 'cloud';
+	};
+
+	/**
+	 * If the user is an instance owner in the cloud, it generates an auto-login link to the
+	 * cloud dashboard that redirects the user to the manage page where they can upgrade to a new n8n version.
+	 * Otherwise, it redirect them to our docs.
+	 */
+	const goToVersions = async () => {
+		let versionsLink = versionsStore.infoUrl;
+
+		if (isInstanceOwnerInCloud()) {
+			versionsLink = await generateCloudDashboardAutoLoginLink({
+				redirectionPath: '/manage',
+			});
+		}
+
+		location.href = versionsLink;
+	};
+
+	const goToDashboard = async () => {
```

Comment:
would this be a better fir in the cloud plans store? maybe generateCloudDashboardAutoLoginLink as well?